addons/hr_holidays/static/src/scss/accrual_plan_level.scss

```diff
@@ -46,7 +39,7 @@ $o-hr-holidays-border-color: map-get($grays, '300');
                 @include o-position-absolute;
                 height: 100%;
                 margin-left: 8px;
-                border-left: 1px dashed darken($o-hr-holidays-border-color, 10%);
+                border-left: 1px dashed darken($o-view-background-color, 10%);
```

Comment:
![Capture d’écran 2023-06-26 à 10 34 33](https://github.com/odoo/odoo/assets/80679690/e80cb725-d48e-4c17-89d1-21e78c7bd9e8)
This color doens't seem to work on darkmode. Maybe `gray-200` could work ?